### ISSUE TITLE
Option to automatically break groups of three quavers

### ISSUE THREAD

**Issue by skirby14** [musescore/MuseScore#25114]
### Your idea

Include an way to automatically break the beams of groups of three quavers, in the following manner:
![image](https://github.com/user-attachments/assets/3367b99b-07d7-429a-b4dc-2c14699a93f5) → ![image](https://github.com/user-attachments/assets/8499cb18-22eb-4a6c-9b1d-bc4ee155e126)

![image](https://github.com/user-attachments/assets/28b8a4c5-f6db-483d-bf85-ecc4391ba8f5) → ![image](https://github.com/user-attachments/assets/159006fd-5fb0-4f2c-ade5-3188ebb8cf0d)

This would _not_ apply to compound time, e.g. 6/8, 9/8, etc. - in this case, three quavers definitely _should_ remain beamed together.

It feels like this would belong in time signature properties, along with the default beam groupings.

### Problem to be solved

It is common practice to break groups of three quavers in this manner, in order to avoid visual confusion with triplets. Currently there is no way to do this automatically in MuseScore - it has to be done manually by spotting each occurrence.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by Dima-S-Jr** [comment]
Good idea. But here, in parallel, another question arises: what about groups of three semiquavers? After all, such groups can also visually resemble triplets: 
![Снимок экрана (666)](https://github.com/user-attachments/assets/516d027e-8ef8-4c01-aeee-8cc0c1ab88f3)
The same goes for demisemiquavers, hemidemisemiquavers, etc.: 
![Снимок экрана (667)](https://github.com/user-attachments/assets/0ce34bf6-3b27-45f4-b175-ed5d6425e61d)
To break such notes, as in the case of quavers, would be incorrect from the point of view of grouping according to the metre. I think the correct convention here would be what is described in #21707. Then the beams would be automatically adjusted to the rest, which would not cause confusion as to whether a group of beams is a tuplet or not.
![Снимок экрана (669)](https://github.com/user-attachments/assets/01027fe1-d6a3-43cd-9569-cc9fd3b7f9dd)
![Снимок экрана (668)](https://github.com/user-attachments/assets/8e287ff5-86ec-49c6-a3d2-0c95ccd631e8)
